src/App.tsx

```diff
@@ -15,35 +16,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  Name = 'name',
+  Length = 'length',
+  NoField = '',
```

Comment:
I guess "Default" property would be clearer than NoField.

Reply:
Agreed! Need to fix.